src/backend/marsha/core/serializers/file.py

```diff
@@ -175,13 +175,25 @@ class InitiateUploadSerializer(serializers.Serializer):
 
     filename = serializers.CharField()
     mimetype = serializers.CharField(allow_blank=True)
+    size = serializers.IntegerField()
 
+    def validate(self, attrs):
```

Comment:
As you are checking only one field, you should probably use the [field-level validation](https://www.django-rest-framework.org/api-guide/serializers/#field-level-validation)